docs/observability/custom-agent-traceability.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Instrumenting BeeAI with OpenInference (Python & JavaScript)"
```

Comment:
Do we need to mention Python and Javascript ? It makes the title quite long in the UI.